src/main/java/gov/cabinetoffice/api/repositories/SubmissionJDBCRepository.java

```diff
@@ -42,9 +42,7 @@ public ApplicationListDTO getApplicationSubmissionsByFundingOrganisationId(final
         final SqlParameterSource applicationParameters = new MapSqlParameterSource()
                 .addValue("fundingOrgId", fundingOrgId);
 
-        final String query = new StringBuilder(APPLICATIONS_WITH_SUBMISSIONS_QUERY)
-                .append(AND_FUNDING_ORG_CLAUSE)
-                .toString();
+        final String query = APPLICATIONS_WITH_SUBMISSIONS_QUERY + AND_FUNDING_ORG_CLAUSE;
```

Comment:
Its a personal preference thing but I think this is more readable the other way

Reply:
Agreed, will revert this.